aya/src/lib.rs

```diff
@@ -33,9 +33,10 @@
 //! [async-std]: https://docs.rs/async-std
 
 #![doc(
-    html_logo_url = "https://aya-rs.dev/assets/images/crabby.svg",
-    html_favicon_url = "https://aya-rs.dev/assets/images/crabby.svg"
+    html_logo_url = "https://aya-rs.dev/assets/images/crabby_dev.svg",
```

Comment:
Huh, shouldn't the userspace documentation have a normal crabee?

Reply:
Argh! Good catch.